SCClassLibrary/Common/GUI/PlusGUI/Math/PlotView.sc

```diff
@@ -13,11 +13,13 @@ Plot {
 		if(Platform.hasQt.not) { ^nil; };	// skip init on Qt-less builds
 
 		StartUp.add {
+			var hlHSV = QtGUI.palette.highlight.asHSV;
```

Comment:
@brianlheim QtGUI.palette is still used here, just refactored into a variable.

Reply:
fontColor is hard coded though

Reply:
> fontColor is hard coded though

Not a trivial point. On my system, the base color is dark, so gray 0.3 is likely to be dark text on a dark background.

Reply:
Ah, I see the issue, thanks for spotting it. I'll look into palettes to see if I can come up with something that works across palettes.